### ISSUE TITLE
Display ALT of sun at noon and AZ at sun-rise/set

### ISSUE THREAD

**Issue by xandro0777** [forrestguice/SuntimesWidget#485]
would be nice :) There is already the line "Solar noon" and adding the ALT information to it seems straightforward and does not consume additional space.

Don't see the ideal place for AZ at sunrise/sunset, probably another (optional) line bellow the sunrise/set times?

**Comment by forrestguice** [comment]
The "Sun Position" dialog shows this already, but only for the centered day ("today").

I can see how this would be useful as part of the main view - it would be possible to flip forward/back to see these values change over time. That would be nice - I'm up for adding this as an option.

There is definitely space in the solar noon line for the ALT. To make room for the AZ I think the "Sunrise" and "Sunset" labels could be removed entirely (leaving just the icons). However, I think its important that users see these labels at least once - the option could be disabled by default.

**Comment by xandro0777** [comment]
> The "Sun Position" dialog shows this already, but only for the centered day ("today").

.. I know I found it somewhere but could not remember where, thanks for reminding me;)

> I can see how this would be useful as part of the main view - it would be possible to flip forward/back to see these values change over time. That would be nice - I'm up for adding this as an option.
> 
> There is definitely space in the solar noon line for the ALT. To make room for the AZ I think the "Sunrise" and "Sunset" labels could be removed entirely (leaving just the icons). However, I think its important that users see these labels at least once - the option could be disabled by default.

The ALT@noon seems fairly starightforward, so pick the low hanging fruit first. 

For more space on the screen maybe the "xxx hours in the day" and "xxx of usable light" might be optional as well. I have already disabled anything from astronomical twilight to golden hour anyway as it was just screenclutter for me and now have plenty of room left even on a fairly small screen. 

My guess.. some people prefer the "technical" informations like ALT, AZ and exact times while other people will prefer the twilight and similar but hardly anyone will want both. Instead of all the twilight/golden/blue hour options technically minded people might prefer something like the inclination of the eclipsis at sunrise/set?

If someone would choose to enable every possible option he should get a warning if it doesn't fit into the screen
